features/prompt_memes_c/challenge_promptmeme_deletion.feature

```diff
@@ -126,7 +126,8 @@ Feature: Prompt Meme Challenge
     And I should not see the prompt meme dashboard for "Battle 12"
     And no one should have a claim in "Battle 12"
     And no one should be signed up for "Battle 12"
-  When I go to the collections page
+  When all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.